interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is an extension of javascript that allows the merger of html tags to be utilized within the Javascript enviorment.  Javascript logic can be called within html tags that that will give interactive with the webpage.  React will take an export from one element to render data on the browser.  This element is call the Parent element and will live within you App.js file.  The data being exported must be witin the return statement. If more than one html tag is being used, or components are being called they must be nested with in React Fragments.
 
-Researched answer:
+Researched answer: JSX component represents HTML tag syntax but is not HTML you can put several components together to make a more complex HTML page. JSX, also isn’t a legitimate JS code, it must be compiled into JS with a tool like Babel.  Taking aspects of the Html gives the ability for presentation with html and logic execution with Javascript.
 
-1. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
+2. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: Yet Another Resource Negotiator.  In Test Driven Developement, having the ability to verify code entries to legacy code is needed to mitigate conflict of data when merging to the main body of a product.  Yarn are a collection of files or dependencies needed to run testing software to troubleshoot and strech test code being developed. There are many dependencies needed, so the a package manager is needed to compile all of the data. ~ not sure what files are modifed, lets find out!
 
-Researched answer:
+Researched answer: Yarn is a package manager for Node. Yarn writes its dependencies to a file named package. json when calling yarn in the terminal. Package.json sits in the root folder of your project, and stores the dependencies files into the node_modules folder. It is commonly used when code is checked out for a project for testing or revising. Another reason to pull dependencies is when another developer has added dependencies to the project.
 
 3. What is a React component?
 
-Your answer:
+Your answer: React Components are ways to compartmentalize features that are added to a appication.  This allows additional features to be created, tested, and deployed without disrupting the main body of the application.  Since App.js can be the only component deployed to React, you can refrence the components using a component call nested within the return statement.
 
-Researched answer:
+Researched answer: Instead of building the whole UI under one single file like HTML. A component is used to divides the UI into smaller pieces. The two types of components are functional and class. 
```

Comment:
Love that you mentioned the two types. Definitely something to keep in mind as you may come across both at some point. 👏🏽